crates/swift-integration-tests/src/shared_types/shared_enum.rs

```diff
@@ -9,8 +9,15 @@ mod ffi {
         fn reflect_enum_with_no_data(arg: EnumWithNoData) -> EnumWithNoData;
     }
 
+    extern "Rust" {
+        #[swift_bridge(Equatable)]
+        type Foo;
+        #[swift_bridge(associated_to = Foo)]
+        fn new() -> Foo;
+    }
+
     enum EnumWithUnnamedData {
-        Variant1(String, u32),
+        Variant1(String, Foo),
```

Comment:
Let's use variant names that clearly indicate what we're testing.

Maybe something like:

```rust
enum EnumWithUnnamedData {
    TwoFields(String, u32),
    OneField(Foo),
    NoFields
}
```